test.sh

```diff
@@ -0,0 +1,7 @@
+set -e
```

Comment:
What is that needed for?

Reply:
set e means fail on errors. I was taking security precautions I felt may be helpful. Although removing the test.sh script as we do not need it anymore do we?

Reply:
I meant whole file.

Reply:
yeah removed